pupil-spa/Dockerfile

```diff
@@ -14,6 +14,7 @@ WORKDIR /ng-app
 
 COPY . .
 
+RUN yarn run postinstall
```

Comment:
£1 says this isn't needed :)

Reply:
You owe me £1 😆

Reply:
Goddamit!

Question is - why is post install not firing automatically?

Reply:
I reckon it does but it then copies as bunch of files into another directory `ng-app`.  We could clean that up by only running install once, in the new directory.